.github/workflows/conformance.yml

```diff
@@ -35,12 +35,18 @@ on:
         description: Always upload logs regardless of the test result.
         type: boolean
         default: false
+      runner:
```

Comment:
I saw the workflow title is like `Run conformance test with Antrea 09b23dce6fbc4fba6b651ad3021421165b2e905a and K8s `. It's quite long, maybe we can replace '${{ inputs.antrea-version }}' in line 2 to '${{ inputs.version.substr(0, 7) }}' ?

Reply:
I tend to agree, however there is no good solution IMO:
1) using an arbitrary length could truncate a branch name or a tag, which I think is not ideal. I am also not sure `substr` can be used in this expression context? We can only do limited things in expressions: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions 
2) I wanted to invoke the workflow using the short sha instead of the full sha, which would avoid the issue. However, this is not supported by the `checkout` action: https://github.com/actions/checkout/issues/265

So I think the current version is the best one, but let me know if you think of a good solution.

Reply:
Ah, OK, I thought something like substr() would work in the name. It seems we have to wait until there is a built-in solution from github. Thanks.